relay-server/src/actors/upstream.rs

```diff
@@ -458,6 +483,42 @@ impl UpstreamRelay {
             .spawn(ctx);
     }
 
+    /// Adds a metric for the upstream request.
+    fn meter_result(
+        request: &UpstreamRequest,
+        send_result: &Result<ClientResponse, UpstreamRequestError>,
+    ) {
+        let sc: StatusCode;
+        let (status_code, result) = match send_result {
+            Ok(ref client_response) => {
+                sc = client_response.status();
+                (sc.as_str(), "success")
```

Comment:
_nit_: Inline `client_response.status().as_str()` and remove the outer scope variable.